project/Dependencies.scala

```diff
@@ -154,6 +154,8 @@ object Dependencies {
       val levelDBmultiJVM = "org.iq80.leveldb" % "leveldb" % "0.12" % "optional;provided;multi-jvm;test" // ApacheV2
       val levelDBNative = "org.fusesource.leveldbjni" % "leveldbjni-all" % "1.8" % "optional;provided" // New BSD
 
+      val lz4Java = "org.lz4" % "lz4-java" % "1.7.1" % "optional;provided;test" // ApacheV2
```

Comment:
This should not be optional/provided because we require it in JacksonSerializer even if it's not enabled.

Reply:
Ah, that was from my feedback, sorry I didn't think about that.